lib/pavilion/plugins/sched/slurm.py

```diff
@@ -307,6 +307,73 @@ def slurm_states(state):
     return states
 
 
+def get_slurm_elems():
```

Comment:
You've got the right idea here, but make keep this as a private, static '_get_config_elems()' method instead.